<launch>

    <arg name="auv_name" value="alpha_rise"/>
    <arg name="asv_name" value="wamv"/>

    <node ns="$(arg asv_name)" pkg="mvp_localization_utilities" type="world_to_world_transform_node" name="world_to_world_tf" output="screen">
        <param name = "auv_tf_prefix" value="$(arg auv_name)"/>
        <param name = "asv_tf_prefix" value="$(arg asv_name)"/>
        <!-- <param name = "mag_declination" value="0.2" /> -->
        <param name = "acceptable_var" value = "5" />
        <param name ="m_gps_wait_time" value ="5"/>
        <!-- beach_pond -->
        <!-- <param name = "datum_latitude" value="41.574236666666664"/> -->
        <!-- <param name = "datum_longitude" value="-71.787575"/> -->
        <!-- allen harbor -->
        <!-- <param name = "datum_latitude" value="41.621601667"/> -->
        <!-- <param name = "datum_longitude" value="-71.411018333"/> -->
        <param name = "datum_latitude" value="41.6216"/>
        <param name = "datum_longitude" value="-71.41095"/>
        <param name = "position_accuracy" value="3.0"/>
        <remap from = "odometry" to ="odometry/filtered/local" />
    </node>



</launch>